### ISSUE TITLE
Use GitHub issues to track Change Control issues

### ISSUE THREAD

**Issue by adrianhopebailie** [mojaloop/mojaloop-specification#9]
As discussed on the CCB call on 19 March:

I propose that we use this issue list as a mechanism fro tracking issues and change requests going forward.

Based upon the current process I recommend the following new process:

  1. Any issues with the current API or related documents is logged in this issue list.
  2. The issue is discussed on the issue list between all stakeholders and if necessary is added to the agenda of CCB calls for deeper discussion (at the discretion of the chair).
  3. If there is consensus that the issue warrants a change to the specification, the change is proposed in the form of a Pull Request
  4. Further discussion and updates to the PR will result in the PR being accepted by the CCB (and the changes merged into the repo) or rejected and the PR closed.

This allows future discussions to easily reference past issues and PRs where context is required.

5. I further propose that a continuous integration process is adopted that produces well-formatted human readable documents from the source material (open api specs and related markdown documents) automatically whenever changes are merged into the master branch of this repo.

6. Finally, the master branch should be protected to not allow changes to be merged without approval from at least a majority of the CCB.

**Comment by elnyry** [comment]
Thanks @adrianhopebailie , I think we are heading this way.

However, as you touched on the topic of readable documents in #5, we're blocked on #3 until thats ready, unless the change is to the Open API Specification itself. However, that still needs to be accompanied by changes in the Spec, so yes we need this before we can go all-in on the PR process.

I've made changes for #6 (also numbered items #5, #6 in your comment to easily reference them)

**Comment by elnyry** [comment]
@adrianhopebailie I think we can now close this. We are now using GitHub issues here to log issues, change proposals and we also have the API Definition v1.0 now available as a markdown doc and published to GitBooks (master branch is protected as well and CI available for releases -> publishes markdown docs to GitBooks)

If you disagree, we can re-open and discuss.
